charts/xui-mo-webapp/values.aat.template.yaml

```diff
@@ -4,6 +4,17 @@ nodejs:
   environment:
     IDAM_URL: https://idam-api.aat.platform.hmcts.net
     S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    NODE_TLS_REJECT_UNAUTHORIZED: 1
     S2S_SECRET: ${S2S_SECRET}
     IDAM_SECRET: ${IDAM_SECRET}
-    NODE_TLS_REJECT_UNAUTHORIZED: 1
+    CCD_DATA_API_SERVICE: http://ccd-data-store-api-demo.service.core-compute-demo.internal
```

Comment:
need to move these urls to values.yaml in a generic way

Reply:
For later as agreed